src/app.js

```diff
@@ -29,6 +29,8 @@ app.use(cookieSession({
 
   // Cookie Options. httpOnly is set by default to true for https
   sameSite: 'lax',
+  secure: true,
```

Comment:
Is this going to interfere running the backend locally (without ssl)?

Reply:
I don't think it is an issue, but I'll remove it anyways since all we really need might be the proxy setting.